raven/contrib/django/models.py

```diff
@@ -237,13 +237,14 @@ def install_middleware():
                     type(middleware)((name,)) + middleware)
 
 
-if (
-    'raven.contrib.django' in settings.INSTALLED_APPS or
-    'raven.contrib.django.raven_compat' in settings.INSTALLED_APPS
-):
+def initialize():
```

Comment:
Maybe this should make sure it only runs once? I am worried this wraps the WSGI and other things twice if run twice.